src/web/layouts/StandardLayout.tsx

```diff
@@ -559,24 +568,25 @@ export const StandardLayout = ({
             </Section>
 
             <Section
+                data-print-layout='hide'
```

Comment:
do you need to repeat data-print-layout='hide' on child DOM nodes?

Reply:
I wasn't sure as it was outside of a div, I'll give it a try. I've already removed some of the nesting